docs/triggertemplates.md

```diff
@@ -1,14 +1,29 @@
 <!--
 ---
-linkTitle: "Trigger Templates"
+linkTitle: "Creating a TriggerTemplate"
 weight: 3
 ---
 -->
-# TriggerTemplates
+# Creating a `TriggerTemplate`
```

Comment:
I don't know if this should be called Creating a "TriggerTemplate". IMO this doc is more of a reference on TriggerTemplates -- all the various fields it can accepts, how it works, etc. Creating a TriggerTemplate sounds more like a subset of this doc.

Reply:
We can call it "Reference: `TriggerTemplate`" - I just didn't think a single-word title was as informative.

Reply:
We've been following the pattern of just using the name of the type in other places, so could we not do the same here and just call it `TriggerTemplates` e.g.  [`TaskRuns`](https://github.com/tektoncd/pipeline/blob/master/docs/taskruns.md)

Reply:
No problem, reverted.